source/user-manual/reference/ossec-conf/wodle-azure-logs.rst

```diff
@@ -475,7 +475,7 @@ Identifier of the application that we will use for the authentication and to be
 graph\\application_key
 ^^^^^^^^^^^^^^^^^^^^^^
 
-Key to the application we will use for authentication and to be able to use the AAD Graph API. It must be used next to the ``application_id`` option obligatorily. Incompatible with ``auth_path`` option.
+Key to the application we will use for authentication and to be able to use the Azure Active Directory Graph API. It must be used next to the ``application_id`` option obligatorily. Incompatible with ``auth_path`` option.
```

Comment:
As I mentioned in the PR description, the Azure Active Directory Graph API is still operational and it is different from the Microsoft Graph API